doc.go

```diff
@@ -1,45 +1,48 @@
-// Copyright 2021 Contributors to the Veraison project.
+// Copyright 2021-2022 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
 
 /*
-Package psatoken provides an implementation of draft-tschofenig-rats-psa-token-08
+Package psatoken provides an implementation of the following two PSA attesation
+token profiles: PSA_IOT_PROFILE_1 and http://arm.com/psa/2.0.0
 
 Creating a PSA Token
 
-The creation of a PSA Token comprises the following steps:
+The creation of a PSA Token with profile http://arm.com/psa/2.0.0 comprises the
+following steps (error checking omitted for brevity):
 
-1. Populating a psatoken.Claims object with the expected claims:
+1. Create a claims-set with the desired profile:
 
-	claims := psatoken.Claims{
-		Profile: psatoken.PSA_PROFILE_2,
-		Nonce: nonce,
-		// see psatoken.Claims{} for other claims
-	}
+	claims, _ := NewClaims(PsaProfile2)
```

Comment:
It is not clear to the first time reader here, that either you have to set 
PsaProfile2 := http://arm.com/psa/2.0.0

Or better state that:
claims, _ := NewClaims('http://arm.com/psa/2.0.0')

Reply:
`PsaProfile2` is a public constant defined by the package.  It's usable as-is.